docs/src/pages/guides/server-rendering/server-rendering.md

```diff
@@ -112,7 +112,8 @@ We will see how this is passed along in the `renderFullPage` function.
 import express from 'express';
```

Comment:
Looks like this page is outdated now

Reply:
Right, should we drop it? We should maybe just change to content to reflect the new styled engine

Reply:
We could do this, maybe in a follow-up?

Reply:
Leaving this comment open for visibility then